store/store.go

```diff
@@ -1190,6 +1200,15 @@ type buyError struct {
 	ErrorMessage string `json:"error_message"`
 }
 
+type storeError struct {
```

Comment:
Right now, I've not actually seen any of the errors that come out of the new endpoints. So I don't do a lot with them yet. We basically just append the error message to an "unexpected HTTP status %d: %s" general purpose error.